main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -560,7 +560,7 @@ public void onCreate(final SQLiteDatabase db) {
             db.execSQL(dbCreateLogsOffline);
             db.execSQL(dbCreateTrackables);
             db.execSQL(dbCreateSearchDestinationHistory);
-
+            db.execSQL(dbCreateTrailHistory);
```

Comment:
Only a minor thing, but I would keep the blank line between the table creation statements and the index call.

Reply:
done